spiketools/sim/neurons.py

```diff
@@ -0,0 +1,118 @@
+"""Simulate cells."""
```

Comment:
In the scope of spiketools (which covers single-neuron analyses and simulations", I don't think "sim/neurons" is specific enough a name - it sounds like simulating neurons, in general (maybe even something like a Hodgkin-Huxley model), but what it actually is is simulating place cells. I think this needs a more specific name - maybe literally `sim/place_cells.py`?